libraries/standard/mqtt/doc/doxygen/config.doxyfile

```diff
@@ -0,0 +1,2556 @@
+# Doxyfile 1.8.19
```

Comment:
This file is auto-generated by doxygen and contains a few changes. Many open source repos continue to use this file in this format.